src/main/java/org/apache/commons/lang3/reflect/MethodUtils.java

```diff
@@ -792,6 +807,7 @@ private static int distance(final Class<?>[] classArray, final Class<?>[] toClas
      * @throws NullPointerException if the specified method is {@code null}
      * @since 3.2
      */
+    @SuppressWarnings({"index:compound.assignment.type.incompatible", "index:array.access.unsafe.high"}) // #4: getGenericParameterTypes() returns an array of Types that represent the formal parameter types of the method object, hence all instances of a class will have the same length of the array returned, hence,  method.getGenericParameterTypes().length =  m.getGenericParameterTypes().length = parameterTypes.length and i < parameterTypes.length inside the loop
```

Comment:
This comment would be much easier to read if it was split over multiple lines. In general, it's good style to avoid lines longer than about 140 characters.